api_specs/v2/component_analyses.yaml

```diff
@@ -1,17 +1,77 @@
 openapi: 3.0.1
 info:
   title: Analytics Core API Specs for Version 2
-  version: "2"
+  version: '2.1'
 servers:
 - url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
   description: "Dependency Analytics Server"
 security:
 - tokenAuth: []
 paths:
+  /component-analyses/:
+    post:
+      tags:
+        - CA batch call
+      summary: Get security and other details about a software component and recommendations to change to a more secured version in a batched manner.
+      parameters:
+        - name: uuid
+          in: header
+          description: User Identification uuid
+          required: false
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RequestBody'
```

Comment:
Can you give a better name? RequestPackageDetails

Reply:
Ack